### PR TITLE
In 150497580 identify sealed sensitive people

### PR DESCRIPTION
**Pull request by AmanShurafa**

### CODE REVIEW

File: spec/features/screening/participant/create_participant_spec.rb [hunk 1]
```diff
@@ -85,7 +87,7 @@ def build_participant_from_person_and_screening(person, screening)
     visit edit_screening_path(id: existing_screening.id)
   end
 
-  scenario 'adding an unknown participant when autocompleter contains results' do
+  scenario 'creating an unknown participant when autocompleter contains results' do
```
Comment (ThomasOSI) [inline spec/features/screening/participant/create_participant_spec.rb]: 👍

File: spec/javascripts/utils/participantHelperSpec.js [hunk 2]
```diff
@@ -197,3 +197,17 @@ describe('nameFormatter', () => {
...
+describe('restrictionFlag', () => {
+  it('returns undefined when no flags are set', () => {
+    expect(restrictionFlag(Immutable.fromJS({}))).toEqual(undefined)
+  })
+
+  it('returns Sealed when both sealed and sensitive flags are set', () => {
+    expect(restrictionFlag(Immutable.fromJS({sealed: true, sensitive: true}))).toEqual('Sealed')
+  })
+
+  it('returns undefined when sensitive flag is set', () => {
```
Comment (ThomasOSI) [inline spec/javascripts/utils/participantHelperSpec.js]: Looks like it returns sensitive...

File: spec/javascripts/utils/participantHelperSpec.js [hunk 3]
```diff
@@ -197,3 +197,17 @@ describe('nameFormatter', () => {
...
+    expect(restrictionFlag(Immutable.fromJS({sensitive: true}))).toEqual('Sensitive')
+  })
+})
```
Comment (ThomasOSI) [inline spec/javascripts/utils/participantHelperSpec.js]: Add test for just sealed and helper will have been unit tested exhaustively and I'll stop complaining about the views.

File: app/javascript/utils/participantHelper.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
 import NAME_SUFFIXES from 'enums/NameSuffixes'
 import _ from 'lodash'
 
+export function restrictionFlag(participant) {
```
Comment (saraOSI) [inline app/javascript/utils/participantHelper.js]: Also, our utils, moving forward, should be using es6 deconstructed parameters, not Immutable objects. You can just call `.toJS()` on the participant when you pass it in.

File: spec/javascripts/components/screenings/participants/ParticipantShowViewSpec.jsx [hunk 5]
```diff
@@ -34,6 +34,31 @@ describe('ParticipantShowView', () => {
...
+    it('renders the sensitive flag', () => {
+      const participant = Immutable.fromJS({
+        ...requiredParticipantProps,
+        sensitive: true,
+      })
+      const component = shallow(<ParticipantShowView participant={participant} onEdit={() => null}/>)
+      expect(component.find('span.information-flag').text()).toEqual('Sensitive')
+    })
+
+    it('renders only the sealed flag even when both are set', () => {
```
Comment (saraOSI) [inline spec/javascripts/components/screenings/participants/ParticipantShowViewSpec.jsx]: This test seems redundant to me. This should be covered in your unit tests.

File: spec/javascripts/components/screenings/participants/ParticipantEditViewSpec.jsx [hunk 6]
```diff
@@ -86,6 +86,29 @@ describe('ParticipantEditView', () => {
...
+    it('renders the sensitive flag', () => {
+      const participant = Immutable.fromJS({
+        ...requiredParticipantProps,
+        sensitive: true,
+      })
+      component = shallow(<ParticipantEditView participant={participant} />)
+      expect(component.find('span.information-flag').text()).toEqual('Sensitive')
+    })
+
+    it('renders only the sealed flag even when both are set', () => {
```
Comment (saraOSI) [inline spec/javascripts/components/screenings/participants/ParticipantEditViewSpec.jsx]: This test seems redundant to me. This should be covered in your unit tests.

**Pull request merged**